replacements/openrct2.music.ragtime/object.json

```diff
@@ -0,0 +1,56 @@
+{
+    "id": "openrct2.music.ragtime",
+    "sourceGame": "official",
+    "authors": ["Karst van Galen Last"],
+    "version": "1.0",
+    "objectType": "music",
+    "properties": {
+        "tracks": [
+            {
+                "source": "music/0.flac",
+                "name": "Searchlight Rag",
+                "composer": "Scott Joplin"
+            },
+            {
+                "source": "music/1.flac",
+                "name": "Entry of the Gladiators",
```

Comment:
Entry of the Gladiators might be better left over for a Circus style, which would potentially fit better under additional (new folder).